.github/workflows/build.yml

```diff
@@ -89,11 +89,21 @@ jobs:
         run: |
             echo $ENCODED_STRING | base64 -di > app/androidkey.jks
 
+      - name: Generate dummy keystore
+        if: ${{ github.event.pull_request.head.repo.fork }}
```

Comment:
Can we rather do this if the secret is empty... That's the true thing you want, no?

Reply:
Makes sense!

Reply:
Changed.

Reply:
Unforty, not worky: 
```
Invalid workflow file: .github/workflows/build.yml#L93
The workflow is not valid. .github/workflows/build.yml (Line: 93, Col: 13): Unrecognized named-value: 'secrets'. Located at position 1 within expression: secrets.KEYSTORE == ''
```
Changed back.

Reply:
You gotta set it to an env and then conditional on the env

eg: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets

Reply:
Oooh, Aaah, I seee, thaanks!